api-operator/deploy/controller-configs/docker_registry_conf.yaml

```diff
@@ -23,4 +23,6 @@ data:
   #docker registry type which the mgw image to be pushed. supported types: DOCKER_HUB, AMAZON_ECR, GCR, HTTP.   Default->  registryType: DOCKER_HUB
   registryType: DOCKER_HUB
   #docker repository name which the mgw image to be pushed.  eg->  repositoryName: docker.io/{USER_NAME of Docker Hub account}
-  repositoryName: REPOSITORY_NAME_OF_DOCKER_REGISTRY
\ No newline at end of file
+  repositoryName: DOCKER_REPOSITORY_NAME
+  #pull secret array
+  imagePullSecretName: docker-my-secret-pull
```

Comment:
shall we make the default value empty

Reply:
and let's add a new line

Reply:
> shall we make the default value empty

sure :)

Reply:
imagePullSecretName -> imagePullSecretNames

Reply:
Updated

Reply:
Fixed via [13add0d](https://github.com/wso2/k8s-api-operator/pull/555/commits/13add0dcfd3e391e2ab922d11a978e285223f363)